src/components/Thought.js

```diff
@@ -459,6 +468,17 @@ const ThoughtContainer = ({
   const styleContainer = getStyle(state, thoughts, { container: true })
   const styleContainerZoom = isEditingPath ? getStyle(state, thoughts.concat('=focus', 'Zoom'), { container: true }) : null
 
+  const cursorOnAlphabeticalSort = cursor && attributeEquals(state, context, '=sort', 'Alphabetical')
+
+  const draggingThoughtPath = pathToContext(state.draggingThought)
```

Comment:
This should be called `draggingThoughtContext` since `pathToContext` returns type `Context`, not type `Path`.